account/manager.go

```diff
@@ -1568,6 +1672,22 @@ func (m *manager) spendAccount(ctx context.Context, account *Account,
 	return account, spendTx, nil
 }
 
+// deriveFeeFromPsbt returns the transaction fee from a given PSBT packet.
+func deriveFeeFromPsbt(packet *psbt.Packet) btcutil.Amount {
```

Comment:
comment: not for this PR but this code is going to be replicated in multiple code repo ([ex in Taro](https://github.com/lightninglabs/taro/pull/184/files#diff-3ef0dca269260a596c1904601701d590c3e180e566016a8de2ced5fb3423fa3cR746). It may be worth adding it to btcd or lnd libs.

```
var packet *psbt.Packet
packet.GetTxFee() 
```

Reply:
I guess we could use the `psbt.SumUtxoInputValues()` function here that is slightly safer as it checks whether there is a witness UTXO. I forgot I added that a while ago :sweat_smile: 
But then we have the question of how to handle the error again...

Reply:
btcd PR: https://github.com/btcsuite/btcd/pull/1916